lib/global_registry/base.rb

```diff
@@ -23,6 +24,24 @@ def self.get(params = {}, headers = {})
     def get(params = {}, headers = {})
       request(:get, params, nil, headers)
     end
+    def self.get_all_pages(params = {}, headers = {})
```

Comment:
Ah, OK, that makes sense, I see how it's using the class name in `default_path`.